pipelines/install/03-tests.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- name: run tests
+  hosts:
+    - "{{ forklift_server_name }}"
+  become: true
+  vars_files:
+    - ../vars/install_base.yml
+    - ../vars/forklift_{{ pipeline_type }}.yml
+  roles:
+    - pulp_health_check
```

Comment:
This is in pulp_all_services, so you are running it twice:
https://github.com/pulp/pulp_installer/blob/master/roles/pulp_all_services/meta/main.yml#L24

Reply:
I know, but I did that explicitly to have a "test" step -- to allow adding more tests here later.